lib/ProjectSwitcher/index.js

```diff
@@ -0,0 +1,81 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import Overlay from 'react-bootstrap/lib/Overlay';
+import Popover from '../PopoverWrapper';
+import { Avatar, Icon } from '../';
+
+/**
+ * @usage
+ *
+ * <AvatarWithPopover
+ *  name="Poppy Cox"
+ *  email="poppycox@gmail.com"
+ *  >
+ *   ... popover content
+ * </AvatarWithPopover>
+ */
+class ProjectSwitcher extends Component {
```

Comment:
This feels like it could be a variation on the existing component `DropdownSwitcher`. Is there any technical difference between the 2 or just visual styling?

Reply:
Mainly visual style dropdowns look very different to popovers. For example there is no speech arrow to point to where it came from. We could always swap `DropdownSwitcher` to use `Popover` what are your thoughts @kyleharper

Reply:
`DropdownSwitcher` accepts a `menu` prop. Maybe we could use the `Popover` for the prop and then add a modifier and style from there.

I think we should have everything required to migrate the `ProjectSwitcher` details into `DropdownSwitcher` 😄

Reply:
@kyleharper have merged `ProjectSwitcher` and `DropdownSwitcher`